launcher/package/rpm/MultiMC5.spec

```diff
@@ -23,22 +29,29 @@ mkdir -p %{buildroot}/opt/multimc
 install -m 0644 ../ubuntu/multimc/opt/multimc/icon.svg %{buildroot}/opt/multimc/icon.svg
 install -m 0755 ../ubuntu/multimc/opt/multimc/run.sh %{buildroot}/opt/multimc/run.sh
 mkdir -p %{buildroot}/%{_datadir}/applications
-install -m 0644 ../ubuntu/multimc/usr/share/applications/multimc.desktop %{buildroot}/%{_datadir}/applications/multimc.desktop
+desktop-file-install --dir=%{buildroot}%{_datadir}/applications ../ubuntu/multimc/usr/share/applications/multimc.desktop
+
 mkdir -p %{buildroot}/%{_datadir}/metainfo
-install -m 0644 ../ubuntu/multimc/usr/share/metainfo/multimc.metainfo.xml %{buildroot}/%{_datadir}/metainfo/multimc.metainfo.xml
+install -m 0644 ../ubuntu/multimc/usr/share/metainfo/multimc.metainfo.xml %{buildroot}/%{_metainfodir}/multimc.metainfo.xml
```

Comment:
`%{_metainfodir}` is the recommended macro for `/usr/share/metainfo`    
https://docs.fedoraproject.org/en-US/packaging-guidelines/AppData/

Reply:
As far as I'm aware, Suse systems don't define that macro (at least I had problems getting a package that refers to it to build there)